x-pack/plugins/siem/server/lib/detection_engine/rules/prepackaged_rules/linux_attempt_to_disable_iptables_or_firewall.json

```diff
@@ -0,0 +1,35 @@
+{
+  "description": "Adversaries may attempt to disable the iptables or firewall service in an attempt to affect how a host is allowed to receive or send network traffic.",
+  "index": [
+    "auditbeat-*"
+  ],
+  "language": "kuery",
+  "name": "Attempt to Disable IPTables or Firewall",
+  "query": "event.action:(executed or process_started) and (process.name:service and process.args:stop or process.name:chkconfig and process.args:off) and process.args:(ip6tables or iptables) or process.name:systemctl and process.args:(firewalld and (disable or stop or kill))",
+  "risk_score": 47,
```

Comment:
Just wondering, how are these defined? I see that the user can select whatever risk_score they want when creating a rule. Is this arbitrary at 47, or is there some meaning behind where it came from?

Reply:
they are linked to severity by default.